src/App.js

```diff
@@ -1,5 +1,81 @@
+import { Console } from "@woowacourse/mission-utils";
+
 class App {
-  async run() {}
+  async run() {
+    try {
+      const input = await this.readInput();
+      const result = this.calculate(input);
+      Console.print(`결과 : ${result}`);
+    } catch (error) {
+      Console.print(error.message);
+      throw error;
+    }
+  }
+
+  async readInput() {
+    return await Console.readLineAsync("덧셈할 문자열을 입력해 주세요:\n");
+  }
```

Comment:
```suggestion
  readInput() {
    return Console.readLineAsync("덧셈할 문자열을 입력해 주세요:\n");
  }
```

이 부분은 async, await을 사용 안해도 될 것 같습니다!